source/includes/fundamentals/code-snippets/Filters.java

```diff
@@ -194,6 +211,7 @@ private void setupPointsCollection(){
         
         List<Document> filterdata = new ArrayList<>();
 
+        // Creates a series of documents and inserts them into the "points" collection
```

Comment:
Suggestion:
I think this should be omitted since it's setup code meant only for tech writers